deploy/olm-catalog/akka-cluster-operator/0.0.1/akka-cluster-operator-certified.v0.0.1.clusterserviceversion.yaml

```diff
@@ -1,13 +1,13 @@
 apiVersion: operators.coreos.com/v1alpha1
 kind: ClusterServiceVersion
 metadata:
-  name: akka-cluster-operator.v0.0.1
+  name: akka-cluster-operator-certified.v0.0.1
   namespace: placeholder
   annotations:
     capabilities: Seamless Upgrades
     categories: Application Runtime
     certified: 'false'
-    containerImage: 'lightbend-docker-registry.bintray.io/lightbend/akkacluster-operator:latest'
+    containerImage: 'lightbend-docker-testing-registry.bintray.io/akka-cluster-operator-certified:0.0.4'
```

Comment:
Good question. The test registry image contains the Dockerfile diffs included here. Those changes can be merged into master so that we can publish a single image hash for both community and certified.